code/sorting/src/postman's sort/postman's_sort.c

```diff
@@ -0,0 +1,102 @@
+// C Program to Implement Postman Sort Algorithm
+
+#include <stdio.h>
+
+int main() 
+{
+    int arr[100], arr1[100];
+    int i, j, c = 0, n = 1, maxdigits = 0;
+    printf("Enter size of array :");
+    int count;
+    scanf("%d", & count);
+    printf("Enter elements into array :");
+    for (i = 0; i < count; ++i) 
+    {
+        printf("Element - %d : ", i);
+        scanf("%d", & arr[i]);
+        arr1[i] = arr[i];
+    }
+
+    //Finding the longest digits in the array and no of elements in that digit
+    for (i = 0; i < count; ++i) 
+    {
```

Comment:
The variable c can be declared at the beginning of this loop. Then you won't need the line `c = 0` on line 31. In addition, a better name for the variable would be `numdigits` because it is clearer.